source/features/show-whitespaces.tsx

```diff
@@ -0,0 +1,96 @@
+import './show-whitespaces.css';
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+import onPrFileLoad from '../libs/on-pr-file-load';
+import onNewComments from '../libs/on-new-comments';
+import getTextNodes from '../libs/get-text-nodes';
+
+function showWhiteSpacesOn(line: Element): void {
+	const textNodes = getTextNodes(line);
+
+	for (const textNode of textNodes) {
+		const nodeValue = textNode.textContent!;
+		if (nodeValue.length === 0) {
+			continue;
+		}
+
+		const fragment = document.createDocumentFragment();
+
+		let lastEncounteredCharType;
+		let charType: 'space' | 'tab' | 'other';
+		let node;
```

Comment:
```suggestion
		let node = 'other';
```

This lets you shorten a following condition, I think